README.md

```diff
@@ -1,45 +1,64 @@
+### Who is in the application?
+
+To make an application collaborative you first create a `space`. A space is the virtual area of an application you want to monitor. A space can be anything from a web page, a sheet within a spreadsheet, an individual slide in a slideshow, or the slideshow itself.
+
+Once a space has been defined, users can enter that space and register their details. Subscribing to `membersUpdate` will notify you of when anyone joins or leaves the space. The most common way to display this is an avatar stack.
+
+### Where is each user within the application?
+
+To display the live location of each user within an application you again utilize a space. You can show where a user is by highlighting the UI element that they have selected, showing that a cell within a spreadsheet has been clicked on, or showing cursors moving across a page.
+
+To show users interacting with UI elements or spreadsheet cells, you can set and track locations using the locations API. Subscribing to `locationUpdate` events will notify all subscribers of any changes in user locations.
 
-> If you are interested in seeing new API features that are still under development, [sign up for exclusive early access](https://go.ably.com/spaces-early-access). If you have any feedback or feature requests, [please fill out this form](https://go.ably.com/spaces-feedback) or reach out to us at [beta@ably.com](mailto:beta@ably.com).
+To display cursors for users within a space, use the cursors API. Subscribing to `positionsUpdate` events will update the location of cursors within an application for all users.
 
-## Overview
+### What is everyone doing in the application?
 
-The Ably Spaces SDK provides purpose-built APIs to enable multiplayer collaboration in any web application. You can set up a collaborative "space" and subscribe to different kinds of updates to see who else is in the space with you, their live location within the app and any updates they are making.
+The Collaborative Spaces SDK is built as an extension to the existing [Ably JavaScript SDK](https://github.com/ably/ably-js). This means that it is possible to utilize the existing pub/sub functionality available with Ably in a collaborative application to track what users are doing. For example, when a user is updating the contents of a cell or field you can use a [channel](https://ably.com/docs/realtime/channels) to show the new contents to other users as it's being typed.
 
-It is built as an extension to our [existing JS SDK](https://github.com/ably/ably-js), so you'll need an Ably JS client to be able to use this library. The usage instructions will help you set that up.
+## Development status
 
-### Glossary
+The Collaborative Spaces SDK is currently under development. 
 
-1. Space: A virtual collaborative space where members can collaborate in realtime with each other.
```

Comment:
What's the reason behind removing this glossary?

Reply:
'Space' seemed the only brand new term, or one that wasn't a term that was implementation heavy. I decided to just explain what it was upfront instead. 'Member' seems self-explanatory and 'avatar stack' is very common. I can add it back in if you think it helps, but that was my thought process.